dspace/config/input-forms.xml

```diff
@@ -5884,794 +199,558 @@
-       <stored-value>Independence of events</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>Joint distribution of discrete random variables</displayed-value>
-       <stored-value>Joint distribution of discrete random variables</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>Linear regression</displayed-value>
-       <stored-value>Linear regression</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>Mean and variance of random variables</displayed-value>
-       <stored-value>Mean and variance of random variables</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>Normal distribution</displayed-value>
-       <stored-value>Normal distribution</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>Observational studies and sample surveys</displayed-value>
-       <stored-value>Observational studies and sample surveys</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>One-sample inference for means</displayed-value>
-       <stored-value>One-sample inference for means</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>One-sample inference for proportions</displayed-value>
-       <stored-value>One-sample inference for proportions</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>One-way analysis of variance</displayed-value>
-       <stored-value>One-way analysis of variance</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>Population vs sample; parameters vs statistics</displayed-value>
-       <stored-value>Population vs sample; parameters vs statistics</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>Probability rules</displayed-value>
-       <stored-value>Probability rules</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>Random variables</displayed-value>
-       <stored-value>Random variables</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>Sampling distributions of means</displayed-value>
-       <stored-value>Sampling distributions of means</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>Sampling distributions of proportions</displayed-value>
-       <stored-value>Sampling distributions of proportions</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>Sampling methods</displayed-value>
-       <stored-value>Sampling methods</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>Two-sample inference for means</displayed-value>
-       <stored-value>Two-sample inference for means</stored-value>
-     </pair>
-     <pair>
-       <displayed-value>Two-sample inference for proportions</displayed-value>
-       <stored-value>Two-sample inference for proportions</stored-value>
-     </pair>
+    <pairs>
+	   <displayed-value>Combinatorics >>> Counting >>> Combinations</displayed-value>
+	   <stored-value>Combinatorics >>> Counting >>> Combinations</stored-value>
```

Comment:
Looks like stored value shouldn't have special characters, so let's replace the spaces and greater than signs with underscores. E.g.: Combinatorics_____Counting_____Combinations